### ISSUE TITLE
How to enable bot on public repo in an org?

### ISSUE THREAD

**Issue by porridge** [renovatebot/config-help#121]
I'm a member of the Mesosphere organization on github. Trying to install renovate on http://github.com/mesosphere/dcos-commons I do not see the org:

![image](https://user-images.githubusercontent.com/489420/48117121-7576db80-e268-11e8-8a66-eb51039dc426.png)

This suggests I do not have enough permissions? However when I ask a Mesosphere admin to do the same, they do not see *any option at all*:

![image](https://user-images.githubusercontent.com/489420/48117158-8b849c00-e268-11e8-9f15-10389fa4cf46.png)

What are we missing?

**Comment by rarkins** [comment]
@porridge this is unfortunately outside the app owner's control, but of course I'll work with you until we get it resolved. GitHub have changed a little over time about who in an Org can install an app, so it hasn't always be clear.

For you, that screenshot is most likely because you are not an owner/admin of mesosphere and you are correct to ask an admin.

For the admin, either:
1. That account is *not* actually an admin of any orgs, or..
2. There is a bug in GitHub's interface

Could you ask them to first create a dummy repository in their personal account as it suggests, and then try the same install process again? I wonder if the lack of personal repository is blocking the regular install screen being shown.

**Comment by rarkins** [comment]
@porridge any luck on this?

**Comment by Schlurrlupp-mesosphere** [comment]
Hi @rarkins 
I did setup a dummy repository and could complete the installation. However this only seems to apply to my personal account, not the http://github.com/mesosphere/dcos-commons repository.

![image](https://user-images.githubusercontent.com/43756680/48354781-d3edf080-e692-11e8-9035-83ce040767e3.png)


I am an owner of the Organisation:
![image](https://user-images.githubusercontent.com/43756680/48354724-aacd6000-e692-11e8-8bc4-a69f5f0c642d.png)


Please advise on how to install on an organisations repo.
Thanks!

**Comment by rarkins** [comment]
Thank you @Schlurrlupp-mesosphere for testing that, it's a pity it didn't solve it. I will email the partner engineering support contact I have to raise this as a bug to them, but if you can email github support from the same email address it could help move things along. In summary it seems that "the owner of the mesosphere org on github.com is not able to install an app into it".

In the meantime, could you also try:

- Can you try installing via both https://github.com/apps/renovate and https://github.com/marketplace/renovate? For the latter, you would pick the "Open Source" plan (free) if you get that far.
- Is your account able to install any *other* apps? e.g. https://github.com/apps/hound You don't actually need to install any, just test if you can get further than with Renovate.

**Comment by Schlurrlupp-mesosphere** [comment]
Thanks @rarkins - actually I was able to configure it for the mesosphere/dcos-commons repo now.
The configure button on your first link (https://github.com/apps/renovate) worked 👍 
![image](https://user-images.githubusercontent.com/43756680/48356129-d867d880-e695-11e8-812e-d017ba7cfc67.png)

So I guess it is only a bug (/missing feature?) while setting up Renovatebot that it only offers to be installed to private repos. The Configure button however, leads me here:
![image](https://user-images.githubusercontent.com/43756680/48356250-22e95500-e696-11e8-85e9-50032686ffce.png)


tl;dr:
It works - thanks for your support!

**Comment by rarkins** [comment]
That's great news! Can I conclude that trying via the Marketplace link didn't work, but the Apps link did?

**Comment by Schlurrlupp-mesosphere** [comment]
Yes. At least I don't see where the Marketplace offers an oportunity to chose a not self-owned repo.
